cmd/gossamer/configcmd.go

```diff
@@ -166,82 +161,58 @@ func getDatabaseDir(ctx *cli.Context, fig *cfg.Config) string {
 	}
 }
 
-// createP2PService starts a p2p network layer from provided config
-func createP2PService(fig *p2p.Config, msgChan chan<- []byte) *p2p.Service {
-	srvc, err := p2p.NewService(fig, msgChan)
-	if err != nil {
-		log.Error("error starting p2p", "err", err.Error())
+func setP2pConfig(ctx *cli.Context, fig p2p.Config) p2p.Config {
+	// Bootnodes
+	if bnodes := ctx.GlobalString(utils.BootnodesFlag.Name); bnodes != "" {
+		fig.BootstrapNodes = strings.Split(ctx.GlobalString(utils.BootnodesFlag.Name), ",")
```

Comment:
I keep getting a parsing error when passing multiple bootnodes in from the command line...not sure if I am correctly passing them in or not though.

` err="failed to parse multiaddr \"\": empty multiaddr"`

Command line:

```
gossamer --bootnodes="/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj","/ip4/104.211.48.51/tcp/30363/p2p/16Uiu2HAmJqVCtF5oMvu1rbJvqWubMMRuWiKJtpoM8KSQ3JNnL5Ec", "/ip4/104.211.48.247/tcp/30363/p2p/16Uiu2HAkyhNWHTPcA2dVKzMnLpFebXqsDQMpkuGnS9SqjJyDyULi"
```
Also tried without "" and [].

Reply:
Either everything is in quotes or nothing. This works for me:
`gossamer --bootnodes=/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj,/ip4/104.211.48.51/tcp/30363/p2p/16Uiu2HAmJqVCtF5oMvu1rbJvqWubMMRuWiKJtpoM8KSQ3JNnL5Ec,/ip4/104.211.48.247/tcp/30363/p2p/16Uiu2HAkyhNWHTPcA2dVKzMnLpFebXqsDQMpkuGnS9SqjJyDyULi
`